libs/platform/experience/src/routes.ts

```diff
@@ -18,13 +18,38 @@ export const defaultExperienceRoutes: RouteConfig[] = [
         .resolveToken('USER.AUTHENTICATED')
         .pipe(
           take(1),
-          map((state) => {
-            if (state) {
-              resolve(RouterService).navigate('/');
-            }
+          map((state) => (state ? '/' : !state))
+        ),
+  },
+  {
+    path: '/:page',
+    type: RouteType.Page,
+  },
+  {
+    path: '/*',
+    type: RouteType.NotFound,
+    render: (): TemplateResult =>
+      html`<oryx-heading><h1>Error 404</h1></oryx-heading>
+        <p>Page not found</p>`,
+  },
+];
 
-            return !state;
-          })
+/**
+ * @deprecated: Since 1.1 version will be deleted. Use `experienceRoutes` instead.
```

Comment:
We will remove it in 2.0 most probably. So let's remove the message about deletion.
```suggestion
 * @deprecated: Since 1.1 version. Use `experienceRoutes` instead.
```